pkg/apis/autoscaling/v2beta1/conversion.go

```diff
@@ -212,7 +212,7 @@ func Convert_v2beta1_PodsMetricSource_To_autoscaling_PodsMetricSource(in *autosc
 }
 
 func Convert_autoscaling_ExternalMetricStatus_To_v2beta1_ExternalMetricStatus(in *autoscaling.ExternalMetricStatus, out *autoscalingv2beta1.ExternalMetricStatus, s conversion.Scope) error {
-	if &in.Current.AverageValue != nil {
+	if in.Current.AverageValue != nil {
```

Comment:
The address of the thing will clearly never be nil. However the correct fix is to remove the `if` and just always execute the body. The change here might affect the behavior.

Reply:
(But I am not sure this is worth changing, it may be this way to have some correspondence to some autogenerated code or something)

Reply:
`Current.AverageValue` is a pointer quantity (https://github.com/kubernetes/kubernetes/blob/ceceb68ceeb69be2a8d1e5855faedcf12fed425a/pkg/apis/autoscaling/types.go#L382) so I think you'll still need to check for that to be nil or you'll get what @tedyu described here https://github.com/kubernetes/kubernetes/pull/79856#issuecomment-509302556